phi/tools/confluence_tools.py

```diff
@@ -0,0 +1,139 @@
+from phi.tools import Toolkit
+from phi.utils.log import logger
+from os import getenv
+
+
+try:
+    from atlassian import Confluence
+except ImportError:
+    raise ImportError("`atlassian` not installed.Please install it using `pip install atlassian-python-api`")
+
+
+class ConfluenceTools(Toolkit):
+    def __init__(self, client_id: str, auth_token: str):
+        super().__init__(name="confluence_tools")
+        self.client_id = client_id or getenv("CONFLUENCE_CLIENT_ID")
+        self.auth_token = auth_token or getenv("CONFLUENCE_AUTH_TOKEN")
```

Comment:
add an if condition to check if both the env variables are present. If not then raise an error to add both these env variables